src/core/useCases/feed/feed.spec.ts

```diff
@@ -265,6 +269,42 @@ describe('Feed', () => {
           zoom: defaultLocationState.zoom,
         },
         types: formatFeedTypes(defaultFeedState.filters.actionTypes, false),
+        timeRange: 192,
+      },
+      nextPageToken: undefined,
+    })
+  })
+
+  it(`
+  Given initial state
+  When user init feed (time range is 8 days)
+    And user clicks to update time range from 8 to 1 day
+  Then store should be update with time range = 1 * 24 hours
+    And items should be fetched
+    And should retrieve feed gateway method have been called for 1 * 24 hours
+`, async () => {
```

Comment:
* indentation
* on ne connait pas la valeur par défaut ici, si elle change, la description du test ne sera plus correcte. Il ne faut pas préciser de valeur dont on n'a pas besoin, ici ce qui nous intéresse c'est de savoir que le feed est initialisé, c'est tout.
* `And user clicks update time range from 8 to 1 day`
  => on ne sait pas que le user click, toi tu le sais parce que tu connais l'UI, mais ici il n'y a pas d'UI.
  => on ne sait pas non plus que la valeur précédente c'est 8, on sait juste que c'est la valeur par défaut

```suggestion
  it(`
    Given initial state
    When user init feed
      And user update time range to 1 day
    Then store should be updated with a 24 hours time range
      And items should be fetched once
      And retrieve feed gateway method should have been called with a 24 hours time range and default locations and types 
  `, async () => {
```